tools/plugins/network/src/smokeTest/kotlin/net/corda/cli/plugins/network/MemberLookupTest.kt

```diff
@@ -0,0 +1,233 @@
+package net.corda.cli.plugins.network
+
+import com.fasterxml.jackson.databind.JsonNode
+import net.corda.cli.plugins.network.output.Output
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import net.corda.e2etest.utilities.DEFAULT_CLUSTER
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.JsonNodeFactory
+import net.corda.cli.plugins.network.utils.HoldingIdentityUtils
+import net.corda.membership.lib.MemberInfoExtension
+import net.corda.membership.lib.MemberInfoExtension.Companion.MEMBER_STATUS_ACTIVE
+import net.corda.membership.lib.MemberInfoExtension.Companion.MEMBER_STATUS_SUSPENDED
+import org.junit.jupiter.api.BeforeAll
+import picocli.CommandLine
+import net.corda.v5.base.types.MemberX500Name
+import org.junit.jupiter.api.BeforeEach
+import java.io.File
+
+class MemberLookupTest {
+    companion object {
+        private lateinit var memberLookup: MemberLookup
+        private lateinit var outputStub: OutputStub
+        private lateinit var onboardMgm: OnboardMgm
+
+        private val CLI_PARAMS = arrayOf(
```

Comment:
Passing these in as `CLI_PARAMS[0]`, `CLI_PARAMS[1]` is not very readable. You could simply do -
```suggestion
        private val targetUrl = DEFAULT_CLUSTER.rest.uri
```
and when you execute the commands -
```
        <command>.execute(
            "-h=$holdingIdentity",
            "--target=$targetUrl",
```